### ISSUE TITLE
Orthogroups.csv has messed up the headers of the original fasta files

### ISSUE THREAD

**Issue by sapuizait** [davidemms/OrthoFinder#545]
Hi there

Pretty much what the title says. After running orthofinder on a big dataset of MAGs in order to identify single copy genes and build a phylogeny the names in the Orthofinder.csv are all wrong.

For starters the headers in the csv file are wrong; if each column represents a genome/MAG within that column, even though most times there are the proteins (cds) of the correct MAG/genome, many times there arent! 

I then tried to ignore what the headers in the csv file say and simple pick the rows (orthologs) that interest me (based on the ones that were only once in a single column and in all genomes/MAGs). But when I do that it seems that the fasta headers (protein names in the original files) are also shuffled up! 

here is an example for you; in the attached file (which is part of a much bigger csv) each column represents a genome (or a MAG in my case) and the field before the first underscore is part of the MAG that it belongs to. As you will see this is mostly consistent but in many cases they are getting mixed up.

I looked at the log of the orthofinder output and there were no errors, so I am not sure if this is a bug and has it happened to other people too? 

[cskjjcn.csv](https://github.com/davidemms/OrthoFinder/files/6337899/cskjjcn.csv)

**Comment by davidemms** [comment]
Hi

The file should be tab-delimited, I think you'll need to open it using only tabs as delimiters and not commas and then all the columns should appear correctly.

All the best
David

**Comment by sapuizait** [comment]
Dear David

Yes, thats exactly what it was! 
However I was having problems when there were consecutively multiple orthologs missing and therefore the software that I used had the tendency to merge the multiple tabs to a single tab.
I found out that it is much easier when I introduced a string signalling the absence of an ortholog. In my case I used sed to simply insert the "ABSENT" string between \t\t. That made it much easier it for me.

Thanks again for looking into it!
P
